components/Footer/Footer.jsx

```diff
@@ -1,5 +1,66 @@
+import Image from 'next/image'
+import britoFlag from '../../public/images/brito-logo-1.svg'
+import britoWordMark from '../../public/images/brito-logo-2.svg'
+import styles from '@/styles/Footer.module.css'
+import { Icon, Nav } from '@/components'
+
 const Footer = (props) => {
-    const { children } = props
-    return <footer>{children}</footer>
+    const { footerContent, navigation, socialMedia } = props
+
+    return (
+        <>
+            <div className={styles.footerContainer}>
+                <div className={`${styles.footerLogoContainer} flex-column-center`}>
+                    <div className='flex-column-space-between'>
+                        <Image
```

Comment:
This should be a background image

Reply:
Is this highlighted code supposed to be a background image?
I tried to follow the same pattern used in the nav bar and Dercilio has refactored it.
![image](https://user-images.githubusercontent.com/110069912/227098828-0579c8be-8c9a-4a9c-a64f-38d9b94a83c8.png)

Reply:
My bad. Never mind :)